### DISCUSSION TITLE
PO-NUMBER

### DISCUSSION THREAD

**Discussion by DJhova** [po-ui/po-angular#1844] [Assuntos gerais]
Olá. 
Nas versões mais antigas do PO-UI o po-number não tinha esse incrementador embutido:
![image](https://github.com/po-ui/po-angular/assets/11687495/263febd5-a851-46c3-aa9b-3134b3723272)

E pelo que vi hoje não tem como retirar, não tem opção para remover isso.
Não seria uma boa deixar esse incrementador opcional? Pois tem campos que não faz sentido ter isso, por exemplo, número da casa, a pessoa tem que digitar.

**Comment by anliben** [comment]
Boa Tarde @DJhova, tudo bem?

Nosso componente hoje em dia segue o padrão default do html onde ele apresenta as arrows. Mas caso queira retirar pode utilizar algum seletor para o input dentro do componente usando este css

```
/* Chrome, Safari, Edge, Opera */
input::-webkit-outer-spin-button,
input::-webkit-inner-spin-button {
  -webkit-appearance: none;
  margin: 0;
}

/* Firefox */
input[type=number] {
  -moz-appearance: textfield;
}
```
